exporter/humioexporter/README.md

```diff
@@ -0,0 +1,66 @@
+# Humio Exporter
+Exports data to Humio using JSON over the HTTP [Ingest API](https://docs.humio.com/reference/api/ingest/).
+
+Supported pipeline types: traces (with metrics and logs to follow soon)
+
+> :construction: This exporter is currently intended for evaluation purposes only!
+
+## Getting Started
+This exporter requires the following configuration options:
+
+- `ingest_token` (no default): The token that has been issued in relation to the Humio repository to export data into. This token grants write-only access to a single, specific Humio repository. See [Ingest Tokens](https://docs.humio.com/docs/ingesting-data/ingest-tokens/) for more details.
+- `endpoint` (no default): The base URL on which the Humio backend can be reached, in the form `host:port`.
+
+As defined in the [TLS Configuration Settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md#tls-configuration-settings), TLS is enabled by default. This can be disabled by overriding the following configuration options:
+
+- `insecure` (default: `false`): Whether to enable client transport security for the exporter's HTTP connection.
+- `insecure_skip_verify` (default: `false`): Whether to skip verifying the server's certificate chain or not.
+
+In addition, the following global configuration options can be overridden:
+
+- `headers` (no default): A series of additional headers in the form of key-value pairs, which should be attached to every request to Humio. It is not possible to override the `Content-Type` or `Authorization` headers, as these are automatically generated by this exporter.
+- `tags` (no default): A series of key-value pairs used to target specific Data Sources for storage inside a Humio repository. Refer to [Humio Tagging](https://docs.humio.com/docs/parsers/tagging/) for more details.
+- `disable_service_tag` (default: `false`): By default, the service name will be used to tag all exported events in addition to user-provided tags. If disabled, only the user-provided tags will be used. However, at least one tag <u>must</u> be specified.
+
+### Logs
```

Comment:
How about removing this for now, given that the exporter doesn't support it? Otherwise, users might be confused: "is this section outdated, or is the previous one outdated?"